workers_launch_template.tf

```diff
@@ -456,7 +456,10 @@ resource "aws_launch_template" "workers_launch_template" {
           count.index,
         )}-eks_asg"
       },
-      var.tags,
+      { for tag_key, tag_value in var.tags :
+        tag_key => tag_value
+        if ! contains([for tag in lookup(var.worker_groups_launch_template[count.index], "tags", local.workers_group_defaults["tags"]) : tag["key"]], tag_key)
```

Comment:
Can you please exclude the tag `Name` also, like you already did for ASG ?

Reply:
Thanks for the review, Yep, I've added this now :)